doc/nrf/ug_downstreams.rst

```diff
@@ -0,0 +1,44 @@
+.. _downstreams:
+
+Downstream Project History
+--------------------------
+
+The |NCS| contains third party Git repositories which may contain additional
```

Comment:
Maybe be a bit more clear about what those are: open source projects that we use as part of NCS, and give a couple of examples (zephyr, mcuboot)